Jenkinsfile

```diff
@@ -110,7 +110,16 @@ parallel(
 			withCredentials([[$class: 'StringBinding',
 												credentialsId: '995d99a7-466b-4beb-bf75-f3ba91cbbc18',
 												variable: 'CODECOV_TOKEN']]) {
-				sh "docker run -e CODECOV_TOKEN=\${CODECOV_TOKEN} --rm ${test_image} sh -c 'test-cover.sh && curl -s https://codecov.io/bash | bash'"
+				def codecov = "codecov -Z -C ${git_commit} "
+				if (git_branch == "remotes/origin/master") {
+					codecov += "-B master"
```

Comment:
Just curious; running `codecov --help` I see that the `-B/-C/-P/-b/-T` vars are `"automatically detected by popular CI providers"` -- maybe some or all of these might be auto-detected?  (Though perhaps the whole point of this PR was they weren't/were incorrect!)